.github/workflows/check-updates.yml

```diff
@@ -0,0 +1,20 @@
+name: Check if the Open API has changed
+on:
+  schedule:
+    - cron:  '0 */3 * * *'
+
+jobs:
+  check-update:
+    if: github.repository == 'tremendous-rewards/tremendous-node'
```

Comment:
do we need this?

Reply:
It prevents forks from running our workflows accidentally